index.js

```diff
@@ -180,11 +180,7 @@
   //  Ord :: TypeClass
   var Ord = $.TypeClass(
     'sanctuary/Ord',
-    function(x) {
-      var type = _type(x);
-      return type === 'Number' && $.ValidNumber.test(x) ||
-             type === 'String';
-    }
+    R.anyPass([$.String.test, $.ValidDate.test, $.ValidNumber.test])
```

Comment:
To be honest, I forgot about [`S.anyPass`][1]! At this point in the file, though, that function has not yet been defined so we can't reference it. I think sticking with the Ramda equivalent is reasonable under these circumstances.


[1]: https://github.com/plaid/sanctuary#anyPass